backend/src/core/services/auth.service.ts

```diff
@@ -3,19 +3,25 @@ import bcrypt from 'bcrypt'
 import { Request } from 'express'
 import config from '@core/config'
 import { loggerWithLabel } from '@core/logger'
-import { User } from '@core/models'
+import { ApiKey, User } from '@core/models'
 import { validateDomain } from '@core/utils/validate-domain'
 import { ApiKeyService, MailService, RedisService } from '@core/services'
 import { HashedOtp, VerifyOtpInput } from '@core/interfaces'
 import { Transaction } from 'sequelize/types'
 
 export interface AuthService {
   canSendOtp(email: string): Promise<void>
+
```

Comment:
**[nit]** don't think these line breaks are necessary?

Reply:
removed the line breaks! @tan-zx